spec/models/user_spec.rb

```diff
@@ -1,14 +1,21 @@
 require "rails_helper"
 
 RSpec.describe User do
-  let(:valid_user_params) { { email: "bob@example.org", password: "secret" } }
+  let(:valid_user_params) { attributes_for(:user) }
+  let(:valid_user) { create(:user) }
+  let(:valid_wishlist_params) { attributes_for(:wishlist) }
```

Comment:
I mean, the whole point of having factories is to get rid of `let`s like this one.